demonstrations/tutorial_mitigation_advantage.py

```diff
@@ -0,0 +1,276 @@
+question in the field is whether or not noisy devices we have access to `right now` are already 
+useful or can outperform a classical computer for certain tasks. For the latter point, 
+demonstrations of quantum computational advantage have been put forward with the 
+`2019 Sycamore <https://www.nature.com/articles/s41586-019-1666-5>`_
+(Google), the `2020 Jiuzhang <https://www.science.org/doi/10.1126/science.abe8770>`_ (Chinese academy of science)
+and the `2022 Borealis <https://www.nature.com/articles/s41586-022-04725-x>`_ (Xanadu) experiments. 
+These demonstrations, however, are only of limited practical utility.
+
+A new quest has been set out 
+for 'practical' quantum computational advantage. That is, a 'useful' application for which a quantum computer 
+outperforms the best known classical methods. In the 
+new article [#ibm]_ by a team of scientists at IBM, a case is made that with their latest device ``ibm_kyiv``
+comprising 127 qubits and record-breaking coherence times, they can faithfully simulate the time 
+dynamics of a complex quantum many-body system. One of the key achievements of the paper is the 
+successful application of error mitigation on a large system (that is making use of a learned noise model [#PEC]_), 
+and demonstrating that it can yield 
+faithful results even in very noisy scenarios with reasonably deep circuits. 
+
+In this demo we are going
+to explain the error mitigation method used in the paper, show how to reproduce the results in PennyLane
+and discuss their implications.
+
+Problem setting
+---------------
+Before we go into the details of the error mitigation methods, let us briefly summarize the problem 
+setting. The authors of [#ibm]_ are concerned with simulating the time dynamics of the 2D transverse field Ising model  
+
+.. math:: H = -J \sum_{\langle qp \rangle}Z_q Z_p + h \sum_q X_q,
+
+with nearest neighbor interactions (indicated by nearest neighbors :math:`\langle qp \rangle`) matching the topology of their 127-qubit ``ibm_kyiv`` device. 
+The time evolution is approximated by trotterization of the time evolution operator
+
+.. math:: U(T) \approx \left(\prod_{\langle qp \rangle} e^{i \delta t J Z_q Z_p} \prod_{q} e^{-i \delta t J X_q} \right)^{\frac{T}{\delta t}}
+
+for an evolution time :math:`T` and a Trotter step size :math:`\delta t`. That means the circuit of concern here is a 
+series of consecutive :math:`\text{RZZ}(\theta_J)` and :math:`\text{RX}(\theta_h)` rotations. The corresponding 
+angles are related to the physical parameters via :math:`\theta_J = -2J \delta t` and :math:`\theta_h = 2h \delta t````. 
+From here on, we are going to focus just on the values of :math:`\theta_h` and keep :math:`\theta_J=-\pi/2` fixed 
+(in the paper, this is due to the simplification this introduces in the decomposition of the :math:`\text{RZZ}` gate in 
+terms of the required CNOT gates).
+
+Noisy simulation of the circuits
+--------------------------------
```

Comment:
@Qottmann if you want this to be a subsection I suggest

```suggestion
Noisy simulation of the circuits
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

We typically use `=` for page titles, `-` for sections, `~` for subsections (I think?) and `^` for subsubsections.

note that ReST doesn't have a predefined order; you can use _any_ symbol, like `"` or `*`. Everytime it encounters a _new_ underline symbol, it treats it as a new lower section in the hierarchy!

Reply:
🤯 thanks! Wasnt aware of that